test/puppetlabs/http/client/decompress_test.clj

```diff
@@ -31,12 +31,6 @@
       (DeflaterInputStream.)))
 
 (deftest gzip-compress-test
-  (testing "clojure gzip decompression"
-    (let [test-response {:headers {"content-encoding" "gzip"}
-                         :body    (gzip compressible-body)}
-          response (async/decompress test-response)]
-      (is (not (contains? (:headers response) "content-encoding")))
-      (is (= compressible-body (slurp (:body response))))))
```

Comment:
Do we have test coverage somewhere else that clearly validates gzip decompression working?